docs/releases/4.2.txt

```diff
@@ -167,6 +167,13 @@ Forms
 * :class:`~django.forms.ModelForm` now accepts the new ``Meta`` option
   ``formfield_callback`` to customize form fields.
 
+* :class:`~django.contrib.auth.forms.BaseUserCreationForm` replaces the
+  old UserCreationForm.
+
+* :class:`~django.contrib.auth.forms.UserCreationForm` now inherits
+  from BaseUserCreationForm and adds a method to reject usernames
+  that differ only in case.
+
```

Comment:
```suggestion
* The new :class:`~django.contrib.auth.forms.BaseUserCreationForm` is added,
  and is the recommended base class for customizing the user creation form.

* :class:`~django.contrib.auth.forms.UserCreationForm` now inherits from
  :class:`~django.contrib.auth.forms.BaseUserCreationForm`, and implements
  ``clean_username`` to reject usernames that differ only in case.
```